package.json

```diff
@@ -101,22 +108,23 @@
     "@babel/polyfill": "^7.4.4",
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.3.3",
+    "@types/enzyme": "^3.10.3",
+    "@types/jest": "^24.0.22",
+    "@types/react": "^16.9.11",
```

Comment:
Please move to `dependencies`.

Reply:
We had several issues when using "@types/react" as depency in a library. So i think we should keep it as devDependency and peerDependency.